NOTICE.txt

```diff
@@ -1932,6 +1932,220 @@ That's all there is to it!
 
 ---------------------------------------------------------
 
+paramiko 2.6.0 - LGPL-2.1-only
```

Comment:
this will be a manual maintain effort in future right? Is there way to track the route work, such as in doc etc.

Reply:
The content can be generated on ADO. I will add a OneNote about how to do it.